mongodump/metadata_dump.go

```diff
@@ -103,7 +91,7 @@ func (dump *MongoDump) dumpMetadata(intent *intents.Intent, buffer resettableOut
 	}
 
 	// Finally, we send the results to the writer as JSON bytes
-	jsonBytes, err := json.Marshal(meta)
+	jsonBytes, err := bson.MarshalExtJSON(meta, true, true)
```

Comment:
I think so far we've been (arbitrarily) not escaping html when marshaling to extJSON. I'm not exactly sure what the value is either way, but for consistency we may want to not escape here either.

Reply:
I wasn't sure what was correct here. If we haven't been doing it for other tools, I'll change it to false here as well.